src/utils.js

```diff
@@ -45,6 +45,7 @@ const prettyUnit = (u) => {
 const isUndefined = s => s === undefined
 
 export default {
+  C, // for custom plugin
```

Comment:
what's this export for?

Reply:
Tests or user custom plugin.

https://github.com/iamkun/dayjs/pull/851/files#diff-f870cbe6ffaae01f79bb9242311d1f1aR133

If I'm going to write a custom plugin, I have to copy these constants manually before.

Reply:
We better not doing this. The constant object will increase a huge bundle size.

Reply:
OK, I found `constant.js` is preserved in `esm` folder, but unavailable in root dir.